### PR TITLE
GitHub-31646: Fixed secret name and YAML

### LINKED ISSUE #31646: [enterprise-4.7] Issue in file cicd/gitops/configuring-sso-for-argo-cd-on-openshift.adoc
### Which section(s) is the issue in?

Configuring Argo CD OIDC

### What needs fixing?

There's no existing secret called `openshift-gitops-secret` so one needs to be created but the docs show an edit command instead of apply.

Second the example yaml has the word `yaml` in it which means you cannot copy/paste it as a template without removing it.

### PR DESCRIPTION
**Pull request by bergerhoffer**
Fixes #31646

Preview: https://deploy-preview-35366--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-sso-for-argo-cd-on-openshift?utm_source=github&utm_campaign=bot_dp#configuring-argo-cd-oidc_configuring-sso-for-argo-cd-on-openshift

### CODE REVIEW

**Comment by stevsmit** [comment]
This looks good to me, but in Step 1 of your Procedure, should you have a colon instead of a period? In other examples I found (for example: https://deploy-preview-35366--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring_argo_cd_to_recursively_sync_a_git_repository_with_your_application/deploying-a-spring-boot-application-with-argo-cd.html), there is a colon before the second level. Just an observation.

**Comment by Preeticp** [comment]
@iam-veeramalla @shubhamagarwal19 one of the core OCP docs team writers has helped us by creating this PR for an [issue raised here](https://github.com/openshift/openshift-docs/issues/31646) Can you please help review this PR and add an LGTM if things look good to you?

File: modules/gitops-configuring-argo-cd-oidc.adoc [hunk 1]
```diff
@@ -25,16 +25,17 @@ $ echo -n '83083958-8ec6-47b0-a411-a8c55381fbd2' | base64
...
 ----
-yaml apiVersion: v1 
-kind: Secret 
-metadata: name: argocd-secret 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-secret
 data:
   oidc.keycloak.clientSecret: ODMwODM5NTgtOGVjNi00N2IwLWE0MTEtYThjNTUzODFmYmQy …
```
Comment (shubhamagarwal19) [inline modules/gitops-configuring-argo-cd-oidc.adoc:40]: can we get rid of those 3 dots`(ODMwODM5NTgtOGVjNi00N2IwLWE0MTEtYThjNTUzODFmYmQy …)` in the end, after the encoded secret value?

Comment (bergerhoffer) [reply modules/gitops-configuring-argo-cd-oidc.adoc:40]: Sure thing!

**Comment by bergerhoffer** [comment]
@shubhamagarwal19 thanks for the review! I removed the `...` - can you let me know if the rest of the updates look good to you?

Also, can you let me know who I should reach out to for QE on this? Thanks!

**Comment by shubhamagarwal19** [comment]
> @shubhamagarwal19 thanks for the review! I removed the `...` - can you let me know if the rest of the updates look good to you?
> 
> Also, can you let me know who I should reach out to for QE on this? Thanks!

@amitkrout @iam-veeramalla can you have a look at it?

**Review by shubhamagarwal19 — APPROVED** [review]
LGTM

**Comment by amitkrout** [comment]
QE ACK
/lgtm

**Comment by Preeticp** [comment]
@bergerhoffer we have the QE and SME ack, can we go ahead and merge this?

**Pull request merged**